### PR TITLE
Added Mandarin translation for Traits

### PR DESCRIPTION
**Pull request by xingyif**
This is good for me. I'm relearning Mandarin. :)

### CODE REVIEW

**Comment by xingyif** [comment]
@coalchan if you got time, can you please also review mine :)

File: _zh-cn/tour/traits.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: tour
+title: 特质
+
+discourse: true
+
+partof: scala-tour
+
```
[the remark this answers is not in the document]
Comment (xingyif) [reply _zh-cn/tour/traits.md]: added

**Comment by xingyif** [comment]
Thanks. Modified based on comments.

**Comment by jvican** [comment]
@xingyif Thanks for this contribution! CI seems to fail in this PR, and @byeonggukgong fixed it today with https://github.com/scala/docs.scala-lang/pull/1129, would you mind rebasing this on top of master?

**Comment by xingyif** [comment]
@jvican rebased on master. CI still fails. I can look into this after work today.

**Comment by SethTisue** [comment]
if there are still failures after all the merges today, I'll open a ticket

**Comment by SethTisue** [comment]
thank you @xingyif and @coalchan !

**Pull request merged**